packages/frontend/src/components/ReactHookForm/BooleanSwitch.tsx

```diff
@@ -11,10 +11,8 @@ const BooleanSwitch: FC<Props> = ({ switchProps, ...rest }) => (
         {...rest}
         render={(props, { field }) => (
             <Switch
-                inline
-                large
-                innerLabelChecked="Yes"
-                innerLabel="No"
+                onLabel="Yes"
```

Comment:
I think we need to increase the Switch size as I can't read the labels.
Now:
![Screenshot 2023-09-08 at 14 10 30](https://github.com/lightdash/lightdash/assets/9117144/9a31c9eb-6a58-4fbb-922b-53e2730d010a)
Before
![Screenshot 2023-09-08 at 14 12 27](https://github.com/lightdash/lightdash/assets/9117144/2afd2a4e-aadb-4ad3-a743-db65bddb86b7)